src/services/admission/admission-router.test.ts

```diff
@@ -0,0 +1,86 @@
+import { beforeEach, describe, expect, it } from "@jest/globals";
+import Models from "../../database/models.js";
+import { DecisionStatus, UpdateEntries } from "./admission-formats.js";
+import { getAsAttendee, getAsStaff, getAsUser, putAsAttendee, putAsStaff, putAsUser } from "../../testTools.js";
+import { DecisionInfo } from "../../database/decision-db.js";
+
+beforeEach(async () => {
+    Models.initialize();
+});
+
+describe("GET /admission", () => {
+    it("gives forbidden error for user without elevated perms", async () => {
+        const responseUser = await getAsUser("/admission/").expect(403);
+        const responseAttendee = await getAsAttendee("/admission/").expect(403);
+        expect(JSON.parse(responseUser.text)).toHaveProperty("error", "InvalidToken");
+        expect(JSON.parse(responseAttendee.text)).toHaveProperty("error", "InvalidToken");
+    });
+
+    it("should return a list of applicants without email sent", async () => {
+        const response = await getAsStaff("/admission/").expect(200);
+        expect(JSON.parse(response.text)).toHaveProperty("entries");
+        JSON.parse(response.text).entries.forEach((entry: DecisionInfo) => {
+            expect(entry.emailSent).toBe(false);
+        });
+    });
+});
+
+describe("PUT /admission", () => {
+    it("gives forbidden error for user without elevated perms (As User and As Attendee)", async () => {
```

Comment:
Break this into two tests, and make updateData a global variable since you use it in other tests.